### DISCUSSION TITLE
Not starting jobs

### DISCUSSION THREAD

**Discussion by infpico** [swar/Swar-Chia-Plot-Manager#222] [Q&A]
Trying to run it via windows, followed all steps, but when manager is running there are no jobs assigned and it is basically idling... 

I have new windows installation, perhaps it have something to do with rights?
![cfg](https://user-images.githubusercontent.com/36266112/117710960-fbc25e80-b1d2-11eb-8569-85ed4829254a.PNG)
![swar](https://user-images.githubusercontent.com/36266112/117710963-fc5af500-b1d2-11eb-9fb9-d0f8243909d7.PNG)

**Comment by Neosettler** [comment]
I'm getting the same treatment. 

Log file:
Error loading config - using defaults

There is no feedback on what the error is.

**Reply by EclipzzeX** [reply]
Further up in the config.yaml, did you properly specify where the chia.exe location is?

**Reply by infpico** [reply]
Thanks, i have linked wrong chia.exe

**Comment by minhhr** [comment]
Had the same issue. In the config file you have to point to the correct chia.exe in chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\

**Reply by LiMoKuangKe** [reply]
thank you, solve my problem

**Reply by infpico** [reply]
Thank you! solved my problem too, I have linked wrong chia.exe destination.

**Reply by elijahfrg** [reply]
Worked for me as well on Windows

**Comment by Fumo696** [comment]
I am starting on Ubuntu 20.04. 
The chia_location in a config.yaml is 
/home/MyUSER/chia-blockchain/venv/bin/chia

And i got the same screen, like a topic starter: 
no any plots are starting and its only disk info. 

What i have to check more?

**Reply by joncav** [reply]
What I found was that old processes that were killed ended up being defunct. This was causing the "job" process to not kick off. What you can do is check if there is in fact a defunct process by running: ps -ef | grep defunct

You might see something like:
user      3626    3624  0 20:56 pts/2    00:00:00 [chia] <defunct>

The first PID is the defunct PID, the second is the parent. You could try killing that process to clear up the defunct process so that jobs can start.

**Reply by joncav** [reply]
What I described was just a symptom. My issue was that one of my disks wasn't writable by my user. I suggest trying to plot manually with the same drives as your config.yaml and check the output. That should tell you what's going wrong.

**Comment by Wilkugmo** [comment]
Hi, I got the same problem. Manager status is: Running but no plots are being made. I think I got correct chia.exe and I don't know what to do.
https://pastebin.com/EgUpbftV

![sawr](https://user-images.githubusercontent.com/84104969/118048058-ffddb000-b37b-11eb-8a39-46377be10319.PNG)

EDIT:
I deleted farmer and pool public keys any it started working. 
Hope it's not a problem since I use 2 diferent keys (1 for plots 1 for storing chia)

**Reply by andi-christ** [reply]
Same problem - here's the correct path (not add windows user name and change chia version as appropriate:

![image](https://user-images.githubusercontent.com/12095704/121644138-fd36bd80-cad5-11eb-9267-5e5eb55f4f6f.png)
